.github/workflows/issue-labeler.yml

```diff
@@ -0,0 +1,19 @@
+name: Issue Labeler
+on:
+  issues:
+    types: [opened, edited]
+
+permissions:
+  issues: write
+  contents: read
```

Comment:
```suggestion
```
We don't need read permission, as we want to add labels to all issues irrespective of their content.

Reply:
It's fine to leave this.

Reply:
We actually do need this because the action needs to read the issues.